go2rtc_client/rest.py

```diff
@@ -119,20 +119,20 @@ def __init__(self, client: _BaseClient) -> None:
         self._client = client
 
     @handle_error
-    async def list(self) -> dict[str, Stream]:
-        """List streams registered with the server."""
-        resp = await self._client.request("GET", self.PATH)
-        return _GET_STREAMS_DECODER.decode(await resp.json())
-
-    @handle_error
-    async def add(self, name: str, source: str) -> None:
+    async def add(self, name: str, sources: str | list[str]) -> None:
```

Comment:
Needed to move the function `add` before `list` otherwise mypy get confused with the `list[str]` type

Reply:
How is the string type handled for `sources` parameter? Is that a single source? Can that be passed as is?

Reply:
See [`de00a76` (#24)](https://github.com/home-assistant-libs/python-go2rtc-client/pull/24/commits/de00a76578e8c2da39e5b1170b8422f7f3213d10)

It will only add one `src` query param instead of multiple